client/tomcatconf/commands.properties.in

```diff
@@ -0,0 +1,804 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
FYI, this deleted file is still being referenced in one of the Marvin smoke tests test/integration/smoke/test_staticroles.py (refer its setUp class).